tutorials/06_Advanced_ChangeInputs.md

```diff
@@ -24,30 +23,57 @@ the name of the needed input data is constructed. It is checked whether those in
 
 The prepared input data is a compressed tar archive file "`.tgz`", which can be opened with software such as [7-Zip](https://www.7-zip.org/), or in terminal by `tar` and `untar` commands. 
 
-Local Input Data
-==================
-If you like to get the input data on your local machine you need to copy your ssh key (`/home/[your_cluster_user_name]/.ssh/id_dsa`) to your computer and adjust your local **.Rprofile** by adding the following line:
-``` r
-options(remind_repos=list("scp://cluster.pik-potsdam.de/p/projects/rd3mod/inputdata/output"=list(username="[your_cluster_user_name]",ssh_private_keyfile="[path_to_your_local_copy_of_your_key]")))
+## Download Input Data Automatically
+
+If you like to automatically download the input data when running REMIND on your local machine, you need to configure this via environment variables.
+On Linux, add to your `~/.bashrc` file (found in your home directory):
```

Comment:
Why not `~/.Renviron`?  That is not affected by the `.Rprofile` in the REMIND repository, the syntax is identical on all systems, … and historic performance of REMIND team members modifying Windows environment variables has been less then stellar.

Reply:
`~/.Renviron` sounds much better, yes, will update the tutorial accordingly.